pkg/apis/addtoscheme_apps_v1alpha1.go

```diff
@@ -2,9 +2,15 @@ package apis
 
 import (
 	"github.com/3scale/3scale-operator/pkg/apis/apps/v1alpha1"
+	routev1 "github.com/openshift/api/route/v1"
+	consolev1 "github.com/openshift/api/console/v1"
+
 )
 
 func init() {
 	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
-	AddToSchemes = append(AddToSchemes, v1alpha1.SchemeBuilder.AddToScheme)
+	AddToSchemes = append(AddToSchemes,
+		routev1.Install,
```

Comment:
`routev1` is being registered in https://github.com/3scale/3scale-operator/blob/c941a204e884f9df6eb1cccc33928db37a3ca568/cmd/manager/main.go#L116

Any good reason to do here again?

Same goes for `consolev1`. Should be registered in `cmd/manager/main.go` or here?

Reply:
yes, agreed. The code moved to main.go.